### PR TITLE
Add function to find the number of parameters for a model

### PR DESCRIPTION
**Pull request by thequilo**

### CODE REVIEW

File: padertorch/summary/model_info.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        >>> num_parameters(nn.Linear(10, 10))
+        model_parameter_size(total=110, trainable=110)
+        >>> net = nn.Sequential(nn.Linear(10, 10).requires_grad_(False), nn.Linear(10, 10))
+        >>> num_parameters(net)
+        model_parameter_size(total=220, trainable=110)
+    """
+    total_number = 0
+    trainable_number = 0
+    for parameter in module.parameters():
+        size = np.prod(parameter.shape)
```
Comment (boeddeker) [inline padertorch/summary/model_info.py:26]: Why not `parameter.numel()`?

File: padertorch/summary/model_info.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+from collections import namedtuple
+from typing import Tuple
+import numpy as np
+from torch import nn
+
+
+def num_parameters(module: nn.Module) -> Tuple[int, int]:
```
Comment (boeddeker) [inline padertorch/summary/model_info.py:7]: Idea: Add an argument `unit` with the values `unit in ['count', 'size']`? (You may know better names)

File: padertorch/summary/model_info.py (hunk 1 above)
Comment (thequilo) [reply padertorch/summary/model_info.py:26]: Because I didn't know that function

File: padertorch/summary/model_info.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    """
+    result = ModelParameterSize()
+
+    for parameter in module.parameters():
+        size = parameter.numel()
+
+        try:
+            bits = torch.finfo(parameter.dtype).bits
+        except TypeError:
+            bits = torch.iinfo(parameter.dtype).bits
```
Comment (boeddeker) [inline padertorch/summary/model_info.py:57]: Using `parameter.element_size()` is shorter. It also avoids the division by 8.

**Review by boeddeker — APPROVED** [review]

File: padertorch/summary/__init__.py [hunk 4]
```diff
@@ -1,2 +1,3 @@
 from .tbx_utils import *
 from . import tfevents
+from .model_info import *
```
Comment (boeddeker) [inline padertorch/summary/__init__.py:3]: Never use `*` import without defining `__all__`. Can you fix that?

**Review by boeddeker — APPROVED** [review]

**Pull request merged**